### PR TITLE
Add award amount metric to Geckoboard dataset

### PR DESCRIPTION
**Pull request by jacksonj04**
This adds the award amount of claims (to the nearest whole pound) to the dataset which is sent to Geckoboard.

To fit within the Geckoboard constraint of 10 columns per dataset this also rolls `checked` and `check_result` into a single field which represents if a claim is unchecked, approved or rejected. Geckoboard panels will need updating to account for this, but the metrics displayed will be identical.

### CODE REVIEW

File: spec/support/geckoboard_helpers.rb [hunk 1]
```diff
@@ -81,12 +83,12 @@ def request_body_matches_geckoboard_data_for_claims?(request, claims)
...
         "submitted_at" => claim.submitted_at&.strftime("%Y-%m-%dT%H:%M:%S%:z"),
         "passed_checking_deadline" => claim.check_deadline_date.past?.to_s,
-        "checked" => claim.check.present?.to_s,
+        "check" => claim.check.present? ? claim.check.result : "unchecked",
         "checked_at" => (claim.check.present? ? claim.check.created_at : DateTime.parse("1970-01-01")).strftime("%Y-%m-%dT%H:%M:%S%:z"),
-        "check_result" => claim.check.present? ? claim.check.result : "",
         "number_of_days_to_check" => claim.check&.number_of_days_since_claim_submitted,
         "paid" => claim.payment.present?.to_s,
         "paid_at" => (claim.payment.present? ? claim.scheduled_payment_date : DateTime.parse("1970-01-01")).strftime("%Y-%m-%dT%H:%M:%S%:z"),
+        "award_amount" => claim&.award_amount.to_i,
```
Comment (robbiepaul) [inline spec/support/geckoboard_helpers.rb]: By casting to an integer here we'd lose the decimal place.

Comment (jacksonj04) [reply spec/support/geckoboard_helpers.rb]: This is a limitation of [Geckoboard's money field](https://api-docs.geckoboard.com/#money-format).

Comment (jacksonj04) [reply spec/support/geckoboard_helpers.rb]: This is a limitation of [Geckoboard's money format](https://api-docs.geckoboard.com/#money-format).

File: spec/models/claim/geckoboard_dataset_spec.rb [hunk 2]
```diff
@@ -3,6 +3,24 @@
...
+
+  it "correctly converts award amounts to an integer of pennies" do
+    ClimateControl.modify ENVIRONMENT_NAME: "test" do
+      eligible_claim = build(
+        :claim,
+        :submitted,
+        student_loan_plan: :plan_1,
+        eligibility: build(:student_loans_eligibility, student_loan_repayment_amount: 2345.67),
+      )
+      expect(Claim::GeckoboardDataset.new(claims: [eligible_claim]).data_for_claims([eligible_claim]).first[:award_amount]).to eq(234567)
```
Comment (robbiepaul) [inline spec/models/claim/geckoboard_dataset_spec.rb]: Nice to have some specs around the values of the dataset 👍

**Pull request merged**